docker-compose.yml

```diff
@@ -24,6 +24,9 @@ services:
     image: openbuildservice/frontend
     build:
       context: .
+    # Workaround to be able to use our shorthand links to ruby, gem & Co
+    environment:
+      - PATH=$PATH:/home/frontend/bin/
```

Comment:
This rather depends on how you enter the container I guess. But anyway this should be the other way around. 

```
- PATH=/home/frontend/bin/:$PATH
```

Reply:
Okay, changed.